fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/CloudGlobalTransactionMgr.java

```diff
@@ -790,10 +791,13 @@ private void getDeleteBitmapUpdateLock(Map<Long, Set<Long>> tableToParttions, lo
                     cumulativePoints.put(tabletId, respCumulativePoints.get(i));
                 }
             }
+            totalRetryTime += retryTime;
         }
         stopWatch.stop();
-        LOG.info("get delete bitmap lock successfully. txns: {}. time cost: {} ms.",
-                 transactionId, stopWatch.getTime());
+        LOG.info(
```

Comment:
The method is called quite frequently, we'd better to print the log only when necessary.
e.g. 
```
if (totalRetryTime > 0 || stopWatch.getTime() > 20) {
    LOG.info(xxx);
}
```

Reply:
It a good idea, i have already changed the code, take a look please.